packages/next/src/build/webpack/loaders/utils.ts

```diff
@@ -15,6 +15,25 @@ export function isClientComponentModule(mod: {
 
 export const regexCSS = /\.(css|scss|sass)(\?.*)?$/
 
+// This function checks if a module is able to emit CSS resources. You should
+// never only rely on a single regex to do that.
```

Comment:
🙏

Reply:
Can this be made extensible for more tools?